js/sales.js

```diff
@@ -123,6 +123,65 @@
         return dataCell;
     }
 
+    function getNewStore(event){
+        event.preventDefault();
+        const form = event.target;
+        const storeName = form.querySelector('#store-name').value;
+        const minHourlyCustomers = parseInt(form.querySelector('#min-hourly-customer').value);
+        const maxHourlyCustomers = parseInt(form.querySelector('#max-hourly-customer').value);
+        const avgCookiesPerCustomer = parseInt(form.querySelector('#avg-cookies-per-customer').value);
+        const newStore = new CookieStore(storeName, minHourlyCustomers, maxHourlyCustomers, avgCookiesPerCustomer);
+        cookieStoreLocations.push(newStore);
+        addNewStore(newStore);
+        form.querySelector('input[type=\'submit\']').disabled = true;
+        Array.from(form.querySelectorAll('input')).forEach(input => {
+            if(input.value !== 'Enter store'){
+                input.value = '';
+            }
+        });
+    }
+
+    function toggleTotalRowHighlight(){
+        if(cookieStoreLocations.length % 2 === 0){
+            document.querySelector('tfoot').removeAttribute('class', 'highlight');
+        } else {
+            document.querySelector('tfoot').setAttribute('class', 'highlight');
+        }
```

Comment:
Although this function looks good, if you want to get fancy, we could reduce it to two lines:
![Capture](https://user-images.githubusercontent.com/20998142/87713392-f76a1800-c75e-11ea-8da6-8d43561ee063.PNG)

It might be crossing into the territory of "clever is not necessarily better than readable", however. What's happening here is using a ternary to decide what method to call, then using [bracket notation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Property_accessors#Bracket_notation) to dynamically access that method on the returned `HTMLElement`.